redisearch/client.py

```diff
@@ -266,15 +274,33 @@ def create_index(self, fields, no_term_offsets=False,
         - **no_term_offsets**: If true, we will not save term offsets in the index
         - **no_field_flags**: If true, we will not save field flags that allow searching in specific fields
         - **stopwords**: If not None, we create the index with this custom stopword list. The list can be empty
+        - **max_text_fields**: If true, we will encode indexes as if there were more than 32 text fields,
```

Comment:
We don't test for `max_text_fields`.
We can use `FT.INFO` for it. To see the flag exist. Similar to `testAlterValidation` in [test.py](https://github.com/RediSearch/RediSearch/blob/master/tests/pytests/test.py).